context/app/static/js/components/detailPage/visualization/VisualizationShareButton/VisualizationShareButton.tsx

```diff
@@ -1,83 +1,59 @@
-import React, { useReducer, useRef } from 'react';
+import React, { useReducer } from 'react';
 
 import ShareIcon from '@mui/icons-material/Share';
-import Popper from '@mui/material/Popper';
-import Paper from '@mui/material/Paper';
-import MenuList from '@mui/material/MenuList';
-import MenuItem from '@mui/material/MenuItem';
-import ListItemIcon from '@mui/material/ListItemIcon';
-import ClickAwayListener from '@mui/material/ClickAwayListener';
-import Link from '@mui/material/Link';
+import LinkIcon from '@mui/icons-material/Link';
+import EmailIcon from '@mui/icons-material/Email';
 
-import { SecondaryBackgroundTooltip } from 'js/shared-styles/tooltips';
 import useVisualizationStore, { VisualizationStore } from 'js/stores/useVisualizationStore';
-import { WhiteBackgroundIconButton } from 'js/shared-styles/buttons';
 import { useSnackbarActions } from 'js/shared-styles/snackbars';
 import { useTrackEntityPageEvent } from 'js/components/detailPage/useTrackEntityPageEvent';
+import IconDropdownMenu from 'js/shared-styles/dropdowns/IconDropdownMenu';
+
 import { copyToClipBoard, createEmailWithUrl } from './utils';
 import { DEFAULT_LONG_URL_WARNING } from './constants';
 
-import { StyledLinkIcon, StyledTypography, StyledEmailIcon } from './style';
-
 const visualizationStoreSelector = (state: VisualizationStore) => ({
   vitessceState: state.vitessceState,
 });
 
 function VisualizationShareButton() {
-  const [open, toggle] = useReducer((v) => !v, false);
-  const anchorRef = useRef(null);
+  const [, toggle] = useReducer((v) => !v, false);
```

Comment:
I think we can drop the `toggle` here since the state it controls is no longer around (i.e. since `open` is removed)